learning_journal/templates/layout.jinja2

```diff
@@ -14,6 +14,7 @@
       <h1>My Python Journal</h1>
       <section id="content">{% block body %}{% endblock %}</section>
     </main>
-    <footer><p>Created in the UW PCE Python Certificate Program</p></footer>
+    <footer><p>Created in the <span style="color:purple;">U</span><span style='color:gold;'>W</span>
```

Comment:
It's tempting, but avoid inline styles.  They are the very devil himself to override.